training/distributed_training/pytorch/data_parallel/maskrcnn/pytorch_smdataparallel_maskrcnn_demo.ipynb

```diff
@@ -4,26 +4,26 @@
    "cell_type": "markdown",
```

Comment:
<p>Since you are updating the version, update markdown from <code>PyTorch(version 1.8.1)</code> -&gt; <code>PyTorch(version 1.10.2)</code>	(applies to both notebooks in this PR)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>